book/src/anchoredkeys.md

```diff
@@ -0,0 +1,21 @@
+# Anchored keys
+Rather than dealing with just keys, we introduce the notion of an *anchored key*, which is a pair consisting of an origin specifier and a key, i.e.
+
+```
+type AnchoredKey = (Origin, Key)
+type Key = String
+```
+
+An *origin* is a triple consisting of a numeric identifier called the *origin ID*, a string called the *origin name* (omitted in the backend) and another numeric identifier called the *gadget ID*, which identifies the means by which the value corresponding to a given key is produced.
+
+The origin ID is defined to be 0 for 'no origin' and 1 for 'self origin', otherwise it is the content ID[^content-id] of the POD to which it refers. The origin name is not cryptographically significant and is merely a convenience for the frontend.
+
+The gadget ID takes on the values in the following table:
+
+| Gadget ID | Meaning                                                                                   |
```

Comment:
Is the Gadget ID meant to semantically explain how the pod was created?
Or is it about the protocol used to create it?

Previously I thought we could have Pod Class and Pod Type.  The class means how the pod was created semantically (by signing, or by proving).  The type is the protocol used (mock signature, Falcon, etc.) https://github.com/0xPARC/pod2/pull/21#discussion_r1940970093